.github/workflows/build.yml

```
@@ -44,10 +37,14 @@
             sudo apt-get install -y libsoxr-dev
         shell: bash
 
-      - name: Pin raven packages from working directory
-        run: opam pin add ./raven --working-dir --yes
+      - name: Install llvm # required for rune
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)